classes/gui/ScatterView.sc

```diff
@@ -1,46 +1,86 @@
-// 2005 by till bovermann
-//   bielefeld university
+/*********************************************************************
+(C) 2005 Till Bovermann / Bielefeld University
 
+and later...
+
+(C) 2018 Jonathan Reus / IEM Graz
```

Comment:
I'm not so sure if (C) is the right term... maybe make it
```
// 2005 / 2018 Till Bovermann (Bielefeld University) / Jonathan Reus (IEM Graz) 
```
?

Reply:
this is the header suggested by the GPL authors
see: https://www.gnu.org/licenses/gpl-3.0.en.html
under How to Apply these Terms to your own programs